### PR TITLE
Refactor FS segmentation label selection

### PR DESCRIPTION
**Pull request by SucyLi**
## Fixes

Related to https://github.com/FCP-INDI/C-PAC/issues/1465 by @SucyLi 

## Description

Change tissue label from integer to list data type so that users can easily choose multiple labels. 

For example, in CCS, we use following labels as WM: 2, 41, 7, 46, 251, 252, 253, 254, 255.

Ref: https://github.com/zuoxinian/CCS/blob/master/H1/ccs_01_anatsurfrecon.sh#L64-L65

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [ ] My pull request targets the `develop` branch of the repository. 
- [x] My commit messages follow best practices.
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I updated the changelog.
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no
  visible errors.

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: CPAC/seg_preproc/utils.py [hunk 1]
```diff
@@ -388,40 +388,27 @@ def hardcoded_antsJointLabelFusion(anatomical_brain, anatomical_brain_mask,
...
 
-    right_wm_label : integer
-        label value corresponding to Right White Matter in multiatlas file
+    gm_label : list
+        a list of integer label value corresponding to Left Gray Matter in multiatlas file
 
-    include_ventricles : boolean
-        whether include labels of ventricles in CSF or not
+    wm_label : list
+        a list of integer label value corresponding to Left White Matter in multiatlas file
```
Comment (shnizzedy) [inline CPAC/seg_preproc/utils.py:408]: Not "Left Gray Matter" and "Left White Matter", just "Gray Matter" and "White Matter", right? It looks like this is taking left and right.

File: CPAC/anat_preproc/anat_preproc.py [hunk 2]
```diff
@@ -1168,7 +1168,6 @@ def freesurfer_preproc(wf, cfg, strat_pool, pipe_num, opt=None):
                                                       'wm_mask'],
                                         function=pick_tissue_from_labels_file),
                           name=f'anat_preproc_freesurfer_tissue_mask')
-    pick_tissue.inputs.include_ventricles = True
```
Comment (shnizzedy) [inline CPAC/anat_preproc/anat_preproc.py:1171]: I guess whether to include ventricles is implicit in which labels are chosen? :cool:

**Review by shnizzedy — APPROVED** [review]
Looks good to me. The changes are easier to read than what was there before.

File: CPAC/seg_preproc/utils.py (hunk 1 above)
Comment (SucyLi) [reply CPAC/seg_preproc/utils.py:408]: Yes! Now "Gray Matter" includes "Left Gray Matter" and "Right Gray Matter". I think it's more condensed and users don't have to distinguish left or right labels.

File: CPAC/anat_preproc/anat_preproc.py (hunk 2 above)
Comment (SucyLi) [reply CPAC/anat_preproc/anat_preproc.py:1171]: Yes! Users can add ventricle labels to CSF label list directly.

File: CPAC/seg_preproc/utils.py (hunk 1 above)
Comment (shnizzedy) [reply CPAC/seg_preproc/utils.py:408]: :+1: 

I meant these docstring lines should get rid of the word "Left":

https://github.com/FCP-INDI/C-PAC/blob/e039df0716789adb965d4482cfdfd4c8f7e43519/CPAC/seg_preproc/utils.py#L405

https://github.com/FCP-INDI/C-PAC/blob/e039df0716789adb965d4482cfdfd4c8f7e43519/CPAC/seg_preproc/utils.py#L408

Comment (SucyLi) [reply CPAC/seg_preproc/utils.py:408]: Oh great catch, I'll fix it in feature/ccs directly. Thanks!

**Pull request merged**